README.md

```diff
@@ -1,76 +1,279 @@
+const fromFigmaShadows = {
+    shad20: new DropShadow({ offset: Point2D.make(0, 1), radius: 4, color: Color.rgba(0, 0, 0, 0.25) }),
+    // abbreviated for terseness
+};
 
- * `yarn compile` - compile all TypeScript to JS.
- * `yarn watch` - compile, then watch for changes.
- * `yarn lint` - lint TypeScript sources.
- * `yarn fix` - lint and automatically fix any automatically fixable lint issues found.
- * `yarn test` - run unit/integration tests.
- * `yarn health` - run tests and lint code with machine-readable outputs for CI.
+export const fromFigmaFonts = {
+    Poppins: {
+        Bold: Font.fromFile("assets/FromFigma.figma.contents/fonts/Poppins-Bold.otf"),
+        Regular: Font.fromFile("assets/FromFigma.figma.contents/fonts/Poppins-Regular.otf")
+    }
+};
 
+const fromFigmaTypography = {
+    heading1: new Typograph({ color: Color.rgba(0, 0, 0, 1), font: fromFigmaFonts.Poppins.Bold, fontSize: 18 }),
+    body: new Typograph({ color: Color.rgba(0, 0, 0, 1), font: fromFigmaFonts.Poppins.Regular, fontSize: 12 })
+};
 
-## Licensing
+export const fromFigmaComponentsFiles = {
+    iconFastFoodHotDog: new File({ src: "assets/FromFigma.figma.contents/components/IconFastFoodHotDog.png" }),
+    iconFastFoodHotDog2x: new File({ src: "assets/FromFigma.figma.contents/components/IconFastFoodHotDog@2x.png" }),
+    iconFastFoodHotDog3x: new File({ src: "assets/FromFigma.figma.contents/components/IconFastFoodHotDog@3x.png" }),
+    iconFastFoodHotDog4x: new File({ src: "assets/FromFigma.figma.contents/components/IconFastFoodHotDog@4x.png" }),
+    // abbreviated for terseness
+};
 
-Diez is published under a dual-license model in two editions:  *Community Edition* and *Enterprise Edition*.  This repository represents Community Edition and is [licensed accordingly.](https://github.com/diez/diez/blob/master/LICENSE.md)
+export const fromFigmaComponents = {
+    iconFastFoodHotDog: Image.responsive("assets/FromFigma.figma.contents/components/IconFastFoodHotDog.png", 24, 24),
+    iconBinocular: Image.responsive("assets/FromFigma.figma.contents/components/IconBinocular.png", 24, 24),
+    // abbreviated for terseness
+};
 
-Learn more about Diez licensing [here](https://diez.org/licensing/).
+export const fromFigmaTokens = {
+    colors: fromFigmaColors,
+    gradients: fromFigmaGradients,
+    shadows: fromFigmaShadows,
+    typography: fromFigmaTypography
+};
 
-## Diez Enterprise Edition
+```
 
-Diez Enterprise Edition includes the Diez DocsGen add-on — an additional compiler target that allows you to build any Diez project into up-to-the-moment design language docs.  Customizable, searchable, self-hostable. Perfect for enterprise design systems.
+Extracted tokens are intended to be _imported_ and _referenced_ by other files in your project — then updated at will be designers, and extracted again whenever you want.  As another application of "semantic tokens," this separation of concerns allows you to treat design files as versionable code assets.
```

Comment:
s/"be designers"/"by designers"

Reply:
Thank you — just pushed a fix, and cleaned this up a bit on the way